### PR TITLE
refactor(@desktop/startup): refactor modal to use StatusQ components

### PR DESCRIPTION
**Pull request by PascalPrecht**
This needs https://github.com/status-im/StatusQ/pull/494

### CODE REVIEW

**Review by alexandraB99 — APPROVED** [review]

File: ui/app/AppLayouts/Onboarding/popups/BeforeGetStartedModal.qml [hunk 1]
```diff
@@ -4,85 +4,91 @@ import QtQuick.Layouts 1.14
...
+            anchors.leftMargin: 32
+            anchors.rightMargin: 32
+
+            Item { height: 12;  width: parent.width }
+
+            StatusCheckBox {
+                id: acknowledge
+                objectName: "acknowledgeCheckBox"
+                width: parent.width
+                Layout.preferredWidth: parent.width
```
Comment (borismelnik) [inline ui/app/AppLayouts/Onboarding/popups/BeforeGetStartedModal.qml:36]: `Layout` property working just in `Layout`. Here its useless.

File: ui/app/AppLayouts/Onboarding/popups/BeforeGetStartedModal.qml [hunk 2]
```diff
@@ -4,85 +4,91 @@ import QtQuick.Layouts 1.14
...
+            }
 
-        StatusCheckBox {
-            id: termsOfUse
-            objectName: "termsOfUseCheckBox"
-            Layout.preferredWidth: parent.width
+            StatusCheckBox {
+                id: termsOfUse
+                objectName: "termsOfUseCheckBox"
+                Layout.preferredWidth: parent.width
```
Comment (borismelnik) [inline ui/app/AppLayouts/Onboarding/popups/BeforeGetStartedModal.qml:44]: same as above

Comment (PascalPrecht) [reply ui/app/AppLayouts/Onboarding/popups/BeforeGetStartedModal.qml:44]: Good catch!

**Review by borismelnik — APPROVED** [review]
Great!

**Pull request merged**